pkg/config/config.go

```diff
@@ -67,6 +68,7 @@ func (c *Config) inExistingRules(r *rule.Rule) bool {
 // ConfigureRules adds the config Rules to DefaultRules
 // Configure RegExps for all rules
 // Configure IncludeNote for all rules
+// Filter out any rules that fall under ExcludedCategories
```

Comment:
```suggestion
// Filter out any rules that fall under ExcludeCategories
```
nit: property consistency